app/models/attendee.rb

```diff
@@ -35,13 +35,20 @@ class Attendee < ApplicationRecord
   validates :gender,          :inclusion => {:in => ["m","f"], :message => "is not valid"}, :presence => true
   validates :given_name,      :presence => true
   validates :guardian_full_name, :presence => { :if => :require_guardian_full_name? }
+  validates :local_phone,
+            presence: true,
+            format: { with: PHONE_REGEX, message: "must contain integers only" },
+            length: { is: 10, message: "must contain exactly 10 digits" },
```

Comment:
Are we using phone numbers from the United States only?  We wouldn't have to work with country codes.  We could make this clear on the registration form.

Reply:
I think so.  I will add some text to the form.